src/NewTools-Inspector-Extensions/StringTest.extension.st

```diff
@@ -0,0 +1,8 @@
+Extension { #name : 'StringTest' }
+
+{ #category : '*NewTools-Inspector-Extensions' }
+StringTest >> testStDisplayString [
```

Comment:
Since you are truncating the strings maybe it might be nice to have a test with a big string?

Little helper for such tests: `String loremIpsum: 1001`